### DISCUSSION TITLE
Issue Retrieving SB Campaigns in Europe

### DISCUSSION THREAD

**Discussion by AlexUnitix** [amzn/ads-advanced-tools-docs#163] [General]
Today, we faced an issue with our application when trying to retrieve a list of SB campaigns in Europe using the /sb/v4/ads/list method. While the response is successful, the list appears empty. This problem is new, as everything was functioning correctly yesterday, and our operations in America remain unaffected.

**Comment by jerirose** [comment]
@AlexUnitix This doesn't appear to be a documentation issue, so please submit a ticket to [the Jira service desk](https://amzn-clicks.atlassian.net/servicedesk/customer/portal/2/group/2) if you have not already done so!

We'll convert this issue to a discussion for now, pending any needed changes to the documentation.

Has anyone else encountered this problem?
